build_manpages/build_manpage.py

```diff
@@ -15,6 +15,17 @@
 from .manpage import Manpage
 
 
+# all manpage attributes that can be set via CLI or setup.cfg
+MANPAGE_DATA_ATTRS = (
+    "authors",  # just "author" in setup.cfg, can be specified multiple times
+    "description",
+    "long_description",
+    "prog",
+    "url",
+    "version",
```

Comment:
Can you document this in the README?  Tthere's the foo=baz already, so perhaps just generalize the options.

Reply:
done